src/customize-pop/gnome-tweaks-extensions/gnome-extensions.md

```diff
@@ -0,0 +1,62 @@
+# GNOME Shell Extensions
+
+GNOME shell extensions are features written by third party developers that build upon the GNOME Shell. Extensions are similar to Chrome Extensions or Firefox Addons.
+
+- **GNOME Extensions** is an application for managing these extensions.
+- **Extension Manager** is a utility for browsing and installing extensions.
+
+Both applications provide menus for configuring, disabling, or removing installed extensions.
+
+---
+
+## Installing the GNOME Extensions & Extension Manager Apps
+
+Launch the Pop!\_Shop and type `extensions` then click `Install` for each application.
+
+![Install Extensions App](/images/gnome-tweaks-extensions/install-extensions-app.png)
+
+## Built-In Extensions
+
+Components packaged in Pop!\_OS are listed in the `Built-In` section. Toggle the switches to enable or disable an extension.
+
+![Built-in Ext](/images/gnome-tweaks-extensions/built-in.png)
+
+These features comprise the Computer Operating System Main Interface Components (COSMIC).
+
+| Extension | Description |
+|-----------|-------------|
+| [Cosmic Dock](https://github.com/pop-os/cosmic-dock) | A modified version of Dash to Dock with different defaults. |
+| [Cosmic Workspaces](https://github.com/pop-os/cosmic-workspaces) | Pop!\_OS's workspaces feature with vertical stacking. |
+| [Pop COSMIC](https://github.com/pop-os/cosmic) | The collection of unique components that comprise the Pop!\_OS experience: Customized Dash-to-Dock, vertical workspaces, and additional configuration options in Settings. |
+| [Pop Shell](https://github.com/pop-os/shell) | A keyboard-driven layer for GNOME Shell which allows for quick and sensible navigation and management of windows. |
+| [System76 Power](https://github.com/pop-os/gnome-shell-extension-system76-power) | A GNOME Shell extension that adds graphical integration to the system76-power daemon. |
```

Comment:
I don't think this is technically considered part of COSMIC-- we ship it on Ubuntu too. It's not bad to list it, but other extensions like DING would make just as much sense to list, since they affect the interface more.

Reply:
Added DING to this table.